docs/maps/mapml-viewer.md

```diff
@@ -0,0 +1,56 @@
+---
+id: mapml-viewer
+title: <mapml-viewer>
+slug: /maps/mapml-viewer
+---
+
+## Introduction
+
+The `<mapml-viewer>` element is the main element you can use to put a custom Web map on your page.  To create a (really) simple Web map, you might use it like this:
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>A Simple Web Map[tm]</title>
+     <meta charset="utf-8" />
+     <script type="module" src="web-map/mapml-viewer.js"></script>
+     <style>html {height:100%} * {margin:0px;padding:0px} body,mapml-viewer {height:inherit}</style>
```

Comment:
These styles aren't required, can we remove them?

Reply:
Ideally we'd encourage people to use the [pre-styling styles](https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/8723e4d8bad8e3629bc9dca7a6fe81d724e8418f/index-mapml-viewer.html#L34-L45) to avoid layout shifts and <span title="Flash of Unstyled Content">FOUC</span>, and the [`<noscript>` styles](https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/8723e4d8bad8e3629bc9dca7a6fe81d724e8418f/index-mapml-viewer.html#L46-L54) if they want to use fallbacks. But maybe we should deal with that as a separate issue.